apps/server/src/modules/provisioning/strategy/oidc/service/oidc-provisioning.service.ts

```diff
@@ -50,10 +51,10 @@ export class OidcProvisioningService {
 	}
 
 	async provisionExternalUser(externalUser: ExternalUserDto, systemId: EntityId, schoolId?: string): Promise<UserDO> {
-		let roleIds: EntityId[] | undefined;
+		let roleRefs: RoleReference[] | undefined;
```

Comment:
I think about the question if RoleReference is stand alone like it is in this PR, or is this part of one DO that must hold and solve it, to save the strict format of the Do. If yes the service that expose this DO must be solve the mapping.

Reply:
The UserDO already exposes the loaded role references, but in this case we want to explicitly overwrite it here.